README.rst

```diff
@@ -1,7 +1,120 @@
-larray
+LArray
 ======
 
 larray provides a Labelled Array class
 
 .. image:: https://travis-ci.org/liam2/larray.svg?branch=master
     :target: https://travis-ci.org/liam2/larray
+
+
+.. start-install
+
+Installation
+============
+
+Pre-built binaries
+------------------
+
+The easiest route to installing larray is through
+`Conda <http://conda.pydata.org/miniconda.html>`_.
+For all platforms installing larray can be done with::
+
+    conda install -c gdementen larray
+
+This will install a lightweight version of larray
+depending on Nympy and Pandas libraries only.
+This version does not allow you to use the included
+graphical user interface, to make plots or to use
+special I/O functions for easy dump/load from Excel
+or HDF files. Optional dependencies are described
+bellow.
+
+Installing larray with all optional dependencies
+can be done with ::
+
+    conda install -c gdementen larrayenv
+
+
+Building from source
+--------------------
+
+The latest release of LArray is available from
+https://github.com/liam2/larray.git
+
+Once you have satisfied the requirements detailed below, simply run::
+
+    python setup.py install
+
+For non-standard locations, additional build lib & include paths
+can be provided as per-usual at build_ext phase::
+
+    python setup.py build_ext -I/path/to/include -L/path/to/lib
+    python setup.py install
+
+
+Update
+------
+
+If larray has been installed through larray package,
+update is done via ::
+
+    conda update larray
+
+Be careful if you have installed optional dependencies.
+In that case, you may have to update some of them.
+
+If larray has been installed through larrayenv,
+you simply must do ::
+
+    conda update larrayenv
+
+This will update larray and all of its dependencies
```

Comment:
it will not update the dependencies at this point (it was the idea, but it does not seem to work yet). I guess we will have to fix each dependency version in the metapackage for this to actually be the case.